recipes/sparseassembler/meta.yaml

```diff
@@ -25,7 +25,7 @@ requirements:
 
 test:
   commands:
-    - SparseAssembler | grep "Error! Genome size not given." > /dev/null
+    - SparseAssembler #| grep "Error! Genome size not given." > /dev/null
```

Comment:
I fixed it, but it end up [here](https://github.com/bioconda/bioconda-recipes/pull/3240)